### ISSUE TITLE
Issues with .dylibs

### ISSUE THREAD

**Issue by easlweasl** [jbclements/portaudio#13]
I'm running racket from macos 12 on an m1 chip,
using (require portaudio) I encountered this:
```
ffi-lib: could not load foreign library
  path: callbacks.dylib
  system error: dlopen(callbacks.dylib, 0x000A): tried: 'callbacks.dylib' (no such file),
 '/usr/local/lib/callbacks.dylib' (no such file), '/usr/lib/callbacks.dylib' (no such file)...
```
Also the paths didnt seem to match for portaudio.dylib either, for that one, I recompiled portaudio and linked it.

**Comment by FuxkingSCUMRotinHELL** [comment]
you are a scum

**Comment by FuxkingSCUMRotinHELL** [comment]
scum

**Comment by trashgotodie** [comment]
You piece of garbage go to die, garbage

**Comment by trashgotodie** [comment]
@easlweasl You piece of trash, go to die worthless LOSER Elias Lajimi

**Comment by Cuntgotodie** [comment]
@easlweasl burn in hell you fucking shithole, lmfao just jerk off to the ugly whore you found on Meetme at your parents house you piece of garbage

**Comment by Cuntgotodie** [comment]
@easlweasl you piece of garbage ruined my fucking life, ex girlfriend lmfao that’s the worst insult ever, go fuck yourself you fucking nightmare

**Comment by Cuntgotodie** [comment]
@easlweasl yes I’m crazy and I will fucking kil you cunt, lmfao shithole hunting for the next victim in Thailand, go pay for some ladyboy to suck your small cock

**Comment by Cuntgotodie** [comment]
@easlweasl Layla dimah, Ridha Lajimi, Elias Lajimi, Cunt’s family, cunt

**Comment by RidhaLajimi** [comment]
@easlweasl ex girlfriend lmfao, worst insult ever, just go jerking off to the whore you found on Meetme at your parents house or ask some ladyboy to suck your cock in Thailand, you cheap jackass

**Comment by RidhaLajimi** [comment]
@easlweasl you piece of garbage ruined my fucking life, garbage, burn in hell you cunt, Worthless trash

**Comment by LajimiRotinHell** [comment]
@easlweasl  Cunt, burn in hell you fucking nightmare ruined my life, go to die

**Comment by LajimiRotinHell** [comment]
@easlweasl you piece of garbage will pay the price trust me, get hit by a truck in fucking Thailand, you piece of trash!!!!!

**Comment by LajimiRotinHell** [comment]
@easlweasl you fucking manipulator ruined my life, lmfao fucking ugly cunt hunting for the next victim in Thailand, Rotten in hell you Loser, Worthless trash, go to die!!!

**Comment by CarefulBadis** [comment]
Can’t wait to see you soon, Badis:)@easlweasl

**Comment by CarefulBadis** [comment]
@easlweasl You fucking trash ruined my life and you will pay the fucking price trust me 
I don't even have to be there to Fuck you Up Lol
Be careful someone will come bring you a big surprise 
Don't be scared you can Shit in your pants
